atlasdb-plugin/src/main/groovy/com/palantir/atlasdb/AtlasPluginExtension.groovy

```diff
@@ -0,0 +1,5 @@
+package com.palantir.atlasdb
+
+class AtlasPluginExtension {
```

Comment:
I like this idea, but I wonder if people will accidentally shoot themselves in the foot.  Regardless, I'll see if this is an easy change.